ui/src/influxql/ast.js

```diff
@@ -0,0 +1,134 @@
+import _ from 'lodash'
```

Comment:
haha, nope, I wrote it. pairing with goller and watts, of course. we powered through it, took us like, five hours, lol.